### ISSUE TITLE
REPLMaker

### ISSUE THREAD

**Issue by musm** [MasonProtter/ReplMaker.jl#13]
Feel free to close this issue, and I totally get that people opening such issues is usually very annoying. In any case, thought I'd mention the following packages
```
REPLHistory     REPLTreeViews    REPLCompletions  REPLTetris       
```
and  the `REPL` stdlib,  which makes me wonder if it would be a good idea to alias the package name to `REPLMaker`

**Comment by MasonProtter** [comment]
Yeah, I definitely regret naming this one `ReplMaker.jl`. I've been thinking of just forking this, and naming the fork `REPLMaker.jl` and registering that.

**Comment by musm** [comment]
I thought there was another way to go about renaming packages in the `Pkg.jl` documentation, but I couldn't find it. 
I mean it should also be possible to literally just rename this repo as well, if I'm not mistaken then reregister.

**Comment by LilithHafner** [comment]
Yes, rename repo and re-register will work.
